grouper/fe/template_util.py

```diff
@@ -11,22 +11,27 @@ def _make_date_obj(date_obj):
     """Given either a datetime object, float date/time in UTC unix epoch, or
     string date/time in the form '%Y-%m-%d %H:%M:%S.%f' return a datetime
     object."""
+    if date_obj == '' or date_obj is None:
+        return ''
+
     if isinstance(date_obj, float):
         date_obj = datetime.fromtimestamp(date_obj, UTC)
 
-    if isinstance(date_obj, basestring):
-        date_obj = datetime.strptime(date_obj, "%Y-%m-%d %H:%M:%S.%f")
-        date_obj = date_obj.replace(tzinfo=UTC)
+    if isinstance(date_obj, unicode) or isinstance(date_obj, basestring):
```

Comment:
`unicode` is a subclass of `basestring`, so `isinstance(date_obj, basestring)` should be enough